doctr/__main__.py

```diff
@@ -122,6 +122,20 @@ def get_parser(config=None):
     deploy_parser = subcommand.add_parser('deploy', help="""Deploy the docs to GitHub from Travis.""")
     deploy_parser.set_defaults(func=deploy)
     deploy_parser_add_argument = make_parser_with_config_adder(deploy_parser, config)
+
+    _user_fork_parser = subcommand.add_parser('fork-deploy', help="""Deploy the docs to GitHub from Travis to the current user fork""")
```

Comment:
Oh, then it should be called `doctr configure-fork` or something like that. Right now this looks like the `deploy` command, which gets run on Travis only.